Source/BugsnagStacktrace.m

```diff
@@ -0,0 +1,46 @@
+//
+//  BugsnagStacktrace.m
+//  Bugsnag
+//
+//  Created by Jamie Lynch on 06/04/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+#import "BugsnagStacktrace.h"
+#import "BugsnagStackframe.h"
+#import "BugsnagCollections.h"
+#import "BugsnagKeys.h"
```

Comment:
I don't think these two imports are required.

Reply:
Removed the unused imports